actioncable/lib/action_cable/connection/base.rb

```diff
@@ -95,7 +95,12 @@ def transmit(cable_message) # :nodoc:
       end
 
       # Close the WebSocket connection.
-      def close
+      def close(reason:, reconnect: true)
```

Comment:
What about adding a default reason? To leave this code backward compatible

Reply:
How about allowing `nil` instead of having a meaningless default?

Reply:
Makes sense